pdns/ws-auth.cc

```diff
@@ -537,7 +537,10 @@ static void gatherRecords(const string& logprefix, const Json container, const D
   const auto& items = container["records"].array_items();
   for(const auto& record : items) {
     string content = stringFromJson(record, "content");
-    rr.disabled = boolFromJson(record, "disabled");
+    rr.disabled = false;
+    if(!record["disabled"].is_null()) {
+      rr.disabled = boolFromJson(record, "disabled");
```

Comment:
`boolFromJson(record, "disabled", false);`?

Reply:
That will send an APIError when `disabled` is not a bool

Reply:
Right. That's probably a bug in all of the <type>FromJson functions.